ponyfill.ts

```diff
@@ -77,7 +83,8 @@ export default async function registerContentScript(
 		}
 
 		for (const file of js) {
-			void chrome.tabs.executeScript(tabId, {
+			// eslint-disable-next-line no-await-in-loop
+			await asyncExecuteScript(tabId, {
```

Comment:
No need, there's already `chromeP` for this
```suggestion
			await chromeP.tabs.executeScript(tabId, {
```